### PR TITLE
Portfolio Project 1: Mobile version

### PR DESCRIPTION
**Pull request by Devtiwo**
In this project, I :

- created index file and added toolbar and headline.
- add styles to the index file.
- add images folder to hold all images required for the project
- add Github workflow folder that holds the linter file that checks for errors.
- add a README file that describes the project.
- add a gitignore file to hold files that don't need to be uploaded to the repository.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,33 @@
...
\ No newline at end of file
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio
+
+> This project is to build a personal portfolio website.
+
+
+## Built With
```
Comment (Rustamxon7) [inline README.md:9]: - [ ] [OPTIONAL] I would suggest you add a screenshot of your webpage to this page. 😃

**Review by Rustamxon7 — CHANGES_REQUESTED** [review]
Hi @Devtiwo , 

Good job so far!

### HIGHLIGHTS!

- No linters errors.💯

There are some issues that you still need to work on to go to the next project but you are almost there!

giphy


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](<https://microverse.zendesk.com/hc/en-us/articles/1500004088561>) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](<https://airtable.com/shrQAqnBwek5a0O0s>)._

File: style.css [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  background-image: url(images/background.png);
+  background-repeat: no-repeat;
+  background-position: 100% 100%;
+}
+
+.intro {
+  color: #ff6b00;
+  font-size: 56px;
+  margin-bottom: 5px;
+  font-family: 'crete round', sans-serif;
```
Comment (mmsesay) [inline style.css:46]: - [ ] [OPTIONAL] Good job in trying to use the right fonts for the intro text. However, there is a typo. The font family should be `Crete Round` Please update the spelling with the right capitalization.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+.content h3 {
+  font-size: 24px;
+  line-height: 32px;
+  font-weight: bold;
+  font-family: Roboto, sans-serif;
+  margin-bottom: 0;
+}
+
+.content p {
+  font-family: poppins, sans-serif;
```
Comment (mmsesay) [inline style.css:58]: - [ ] [OPTIONAL] Good job in trying to use the right fonts for the intro text. However, there is a typo. The font family should be `Poppins` Please update the spelling with the right capitalization.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+.toolbar img {
+  width: 18px;
+  height: 12px;
+  cursor: pointer;
+}
+
+.toolbar a {
+  color: #fff;
+  text-decoration: none;
+}
```
Comment (mmsesay) [inline style.css:29]: - [ ] [OPTIONAL] Your toolbar `a` tag is missing the correct font family. The font family should be `Roboto` Please try to add it.

**Review by mmsesay — APPROVED** [review]
Hi @Devtiwo , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**